.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDatePeriodParser.cs

```diff
@@ -508,6 +568,19 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
                 er[1].Start -= this.config.TokenBeforeDate.Length;
             }
 
+            var match = this.config.WeekWithWeekDayRangeRegex.Match(text);
+            string weekPrefix = null;
+            if (match.Success)
+            {
+                weekPrefix = match.Groups["week"].ToString();
+            }
+
+            if (!string.IsNullOrEmpty(weekPrefix))
+            {
+                er[0].Text = weekPrefix + " " + er[0].Text;
```

Comment:
Don't concatenate string like this. Use string interpolation for easier to read code.